python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -186,7 +195,7 @@ def _get_metadata_from_chat_response(self, response: ChatCompletion) -> dict[str
             "id": response.id,
             "created": response.created,
             "system_fingerprint": response.system_fingerprint,
-            "usage": response.usage if hasattr(response, "usage") else None,
+            "usage": CompletionUsage.from_openai(response.usage) if response.usage is not None else None,
```

Comment:
Will the `usage` attribute on the response always be present? The reason we had `hasattr(...)` before was because it wasn't always included.

Reply:
Yes, it's a property of the response object.